### PR TITLE
Fix Color Pop-In Bug

### PR DESCRIPTION
**Pull request by justinmc**
See issue: https://github.com/Autodesk/notebook-molecular-visualization/issues/39

### CODE REVIEW

File: nbmolviz/drivers3d.py [hunk 1]
```diff
@@ -71,30 +71,40 @@ def set_background_color(self, color, opacity=1.0):
...
+    def set_clipping(self, near, far):
+        self.viewer('setSlab', [float(near), float(far)])
+
     def set_color(self, color, atoms):
+        self.styles = self.get_styles_for_color(
+                color, atoms, self.styles
+        )
+
+    # Returns new styles after updating the given atoms with the given color
+    def get_styles_for_color(self, color, atoms, styles):
```
Comment (justinmc) [inline nbmolviz/drivers3d.py]: This instance method doesn't use `self`.  Is that usually ok in Python?  Should I make it a class method, or move it to some other file outside of a class altogether?

**Comment by justinmc** [comment]
@avirshup Needs review please!

File: nbmolviz/drivers3d.py (hunk 1 above)
Comment (avirshup) [reply nbmolviz/drivers3d.py]: Yeah, this is fine; it's a actually a `staticmethod` (in python, a `classmethod` is one that gets the class as the first argument). So you can write this as

``` python
@staticmethod
def get_styles_for_color(color, atoms, styles):
```

Comment (justinmc) [reply nbmolviz/drivers3d.py]: Ok got it, I'll make this change really quick because it bothered me.

**Comment by avirshup** [comment]
:+1: fix confirmed, code looks good :+1:

**Pull request merged**